src/e

```diff
@@ -169,7 +169,7 @@ program
   .command('backport [pr]', 'Assists with manual backport processes')
   .command('show <subcommand>', 'Show info about the current build config')
   .command('test [specRunnerArgs...]', `Run Electron's spec runner`)
-  .command('pr [options]', 'Open a GitHub URL where you can PR your changes')
+  .command('pr [subcommand]', 'Open a GitHub URL where you can PR your changes')
```

Comment:
```suggestion
  .command('pr [subcommand]', 'Work with PRs to electron/electron')
```

Alternative wordings welcome, but we should change this to reflect that there's more commands here now.